app/src/main/java/bomb/KamikazeBomb.java

```diff
@@ -0,0 +1,69 @@
+package bomb;
+
+import android.graphics.Canvas;
+import android.graphics.Color;
+import android.graphics.Paint;
+
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.List;
+
+import hacking.to.the.gate.Collider;
+import hacking.to.the.gate.Hittable;
+import jet.FriendJet;
+import jet.JetAnimation;
+
+/**
+ * Created by Ruiqian on 9/20/2015.
+ */
+public class KamikazeBomb extends Bomb{
+
+    private List<FriendJet> mFriendJets;
+    public void onCollision(Hittable h){}
+    public Collider getCollider(){
+        return null;
+    }
+
+    public List<FriendJet> getHittableChildren(){
```

Comment:
Change the signature to List<Hittable> to emphasize the usage of this method. @Jelly-Yu